### ISSUE TITLE
Example in "Read the TOC, MCN and ISRCs" does not work

### ISSUE THREAD

**Issue by thomasregnet** [phw/ruby-discid#18]
I get this error:

```
/home/ossi/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/discid-1.4.0/lib/discid/disc.rb:205:in `new': tried to create Proc object without a block (ArgumentError)
        from /home/ossi/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/discid-1.4.0/lib/discid/disc.rb:205:in `tracks'
        from ./discid.rb:15:in `<main>'
```

Using `disc.tracks.each do |track|` would fix this issue.

**Comment by phw** [comment]
That's probably a Ruby 3 issue. As of now this library is untested on Ruby 3. I'll check what needs to be done to support Ruby 3.

**Comment by phw** [comment]
Should be fixed with d620f9a, also setup CI with Ruby 3. Would be great if you could check whether this fix works for you as well. I'll do a new gem release later this week.

**Comment by phw** [comment]
I published a new version 1.4.1 with the fix for this

**Comment by thomasregnet** [comment]
Works :).

Thanks
